packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -570,16 +572,8 @@ export abstract class FluidDataStoreContext
 	): void {
 		const safeTelemetryProps = extractSafePropertiesFromMessage(message);
 		// On op process, tombstone error is logged in garbage collector. So, set "checkTombstone" to false when calling
-		// "verifyNotClosed" which logs tombstone errors. Throw error if tombstoned and throwing on load is configured.
+		// "verifyNotClosed" which logs tombstone errors.
 		this.verifyNotClosed("process", false /* checkTombstone */, safeTelemetryProps);
```

Comment:
```suggestion
		this.verifyNotClosed("process", true /* checkTombstone */, safeTelemetryProps);
```

Reply:
As the comment says, process results in call to GC which will do the tombstone validation. So, we don't need to do it here. Ideally, as you had suggested somewhere else, we should move the other usages of this to GC as well.

Reply:
Ok. I was thrown off by the fact that on the LHS of this change we were doing something here.